src/winston.js

```diff
@@ -4,27 +4,34 @@
  */
 
 require('le_node');
-var winston = require('winston');
+const winston = require('winston');
 
-module.exports = function(token) {
-  //we do not want console transport in production, because it reduce performances
+module.exports = (notifier, token, notify = true) => {
```

Comment:
You are right, this will be a hassle as most of the errors will be duplicated if we publish services without updating logic code for now

However I think this default behavior is good